lib/connect/wx.js

```diff
@@ -104,6 +104,27 @@ function buildStream (client, opts) {
 
   proxy = buildProxy()
   stream = duplexify.obj()
+  stream._destroy = function (err, cb) {
+    socketTask.close({
+      success: function () {
+        cb && cb(err)
+      }
```

Comment:
You might want to call the callback in case or error as well.

Reply:
sorry, it cause by donot merge code from master.